### DISCUSSION TITLE
[Feature] winget-pkgs pull

### DISCUSSION THREAD

**Discussion by Hofknecht** [Hofknecht/SystemTrayMenu#316] [Ideas]
see
https://www.golem.de/news/microsoft-windows-bekommt-einen-paketmanager-2005-148618.html
https://github.com/microsoft/winget-pkgs/pulls

**Comment by Hofknecht** [comment]
Id: Hofknecht.SystemTrayMenu
Publisher: Hofknecht
Name: SystemTrayMenu
Version: 0.10.1.2
License: GPL-3.0
AppMoniker: SystemTrayMenu
Description: SystemTrayMenu is an improved open source toolbar for Windows.
Homepage: https://github.com/Hofknecht/SystemTrayMenu
InstallerType: nullsoft
Installers:
  - Arch: x64
    InstallerType: nullsoft
    Url: https://github.com/Hofknecht/SystemTrayMenu/releases/latest/download/SystemTrayMenu.zip
    Sha256: 	3b0b944991586c1912fdeb96d61a5222fc1bfb1941a58de4349968f682c2634f

**Comment by Hofknecht** [comment]
[0.10.1.2.zip](https://github.com/Hofknecht/SystemTrayMenu/files/4713138/0.10.1.2.zip)

**Comment by Hofknecht** [comment]
https://www.virustotal.com/gui/file/3b0b944991586c1912fdeb96d61a5222fc1bfb1941a58de4349968f682c2634f/detection

**Comment by Hofknecht** [comment]
compare also against this
https://github.com/microsoft/winget-pkgs/blob/master/manifests/TortoiseGit/TortoiseGit/2.10.0.2.yaml

![image](https://user-images.githubusercontent.com/52528841/83652048-7691fb00-a5ba-11ea-945c-2b09d471f928.png)

**Comment by Tanjalibertatis** [comment]
"Standalone or portable executables, compressed .zip files, and fonts are not currently supported."

=> so we need an installer first
